test/basic.test.js

```diff
@@ -126,18 +126,21 @@ test('basic prettifier tests', (t) => {
       write (chunk, enc, cb) {
         const formatted = pretty(chunk.toString())
         const zonedDateTime = joda.ZonedDateTime.ofInstant(
-          joda.Instant.now(),
+          joda.Instant.ofEpochMilli(Date.now()),
           joda.ZoneOffset.SYSTEM
         )
         const offset = joda.DateTimeFormatter.ofPattern('Z').format(
           zonedDateTime
         )
-        const hour = Math.floor(
-          ((17 * 3600) + joda.ZoneOffset.of(offset).totalSeconds()) / 3600
+        const dateStr = joda.DateTimeFormatter.ofPattern('yyy-MM-dd').format(
+          zonedDateTime
```

Comment:
This, and the hour, should be calculated from the static `epoch` variable. Otherwise, this test is always going to fail because `Date.now()` will always be after the static test data.

Reply:
This should also be `yyyy-MM-dd` (you're missing a `y`).

FYI: I originally calculated from the current `Date.now()` because I thought all I needed was the offset. That is not the case.

Reply:
Not enough clear for me. `zonedDateTime` is create using 
```
joda.ZonedDateTime.ofInstant(
          joda.Instant.ofEpochMilli(Date.now()),
          joda.ZoneOffset.SYSTEM
        )
```

where `Date.now` is mocked using 'epoch` before each test:

```
t.beforeEach((done) => {
    Date.originalNow = Date.now
    Date.now = () => epoch

    done()
  })
```

So, We create zoned instant from `epoch = 1522431328992` and only format it (`dateStr` and `hour`).

Do you think that need change `Date.now` -> `epoch` here `joda.Instant.ofEpochMilli(Date.now())` ?

Reply:
I did forget that `Date.now` is mocked. However, it is mocked so that `pino` returns the correct time. Within the test itself I think it will be clearer to use `epoch`. When I read `Date.now()` I think "okay, the date at the exact time this statement is evaluated" where as if I read a locally defined constant then I know it's just that, a constant.

Reply:
In short, yes, `joda.Instant.ofEpochMilli(Date.now())` should be `joda.Instant.ofEpochMilli(epoch)` in the test.